src/V128.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Windows.Markup;
+
+namespace Wasmtime
+{
+    /// <summary>
+    /// A 128 bit value
+    /// </summary>
+    public struct V128
+    {
+        /// <summary>
+        /// Get a V128 with all bits set to 1
+        /// </summary>
+        public static readonly V128 AllBitsSet = new V128(
+            byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue,
+            byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue,
+            byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue,
+            byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue
+        );
+
+        private unsafe fixed byte Value[16];
+
+        /// <summary>
+        /// Construct a new V128 from a 16 element byte span
+        /// </summary>
+        /// <param name="bytes"></param>
+        /// <exception cref="ArgumentException"></exception>
+        public V128(ReadOnlySpan<byte> bytes)
+        {
+            if (bytes.Length != 16)
+                throw new ArgumentException("Must supply exactly 16 bytes to construct V128");
+
+            unsafe
+            {
+                fixed (byte* value = Value)
+                    bytes.CopyTo(new Span<byte>(value, 16));
+            }
+        }
+
+        /// <summary>
+        /// Construct a new V128 from 16 bytes
+        /// </summary>
+        /// <param name="e0"></param>
```

Comment:
Nit: perhaps these should be `b0` to `b15`? Also, for completion, could we fill in these param comments, even if it's a rather meaningless "The first byte value", "The second byte value", etc.?

Reply:
That's a much better name. Honestly not sure what I was htinking of with `e`...